DigitalLearningSolutions.Data.Tests/Services/RegistrationServiceTests.cs

```diff
@@ -1126,6 +1164,62 @@ public void CreateDelegateAccountForExistingUser_with_unapproved_IP_registers_de
             approved.Should().BeFalse();
         }
 
+        [Test]
+        public void CreateDelegateAccountForExistingUser_adds_new_delegate_to_groups()
+        {
+            // Given
+            const int userId = 2;
+            const int delegateId = 2;
+            var model = RegistrationModelTestHelper.GetDefaultInternalDelegateRegistrationModel();
+            var delegateEntity = UserTestHelper.GetDefaultDelegateEntity(
+                7,
+                answer1: "a",
+                answer2: "b",
+                answer3: "c",
+                answer4: "d",
+                answer5: "e",
+                answer6: null
+            );
+            var expectedAnswers = delegateEntity.GetRegistrationFieldAnswers();
+            A.CallTo(
+                () => registrationDataService.RegisterDelegateAccountAndCentreDetailForExistingUser(
+                    A<DelegateRegistrationModel>._,
+                    A<int>._,
+                    A<DateTime>._,
+                    A<IDbTransaction?>._
+                )
+            ).Returns((delegateId, "fake"));
+            A.CallTo(() => userDataService.GetDelegateById(delegateId)).Returns(delegateEntity);
+
+            // When
+            var (_, approved, _) = registrationService.CreateDelegateAccountForExistingUser(

```

Comment:
You can delete `var (_, approved, _) = `

Reply:
Isn't it needed for the assertion on line 1125?

Reply:
No...this is line 1195, in test `CreateDelegateAccountForExistingUser_adds_new_delegate_to_groups`. The assignment for 1125 is on 1099 (`CreateDelegateAccountForExistingUser_on_localhost_registers_delegate_as_approved`).